multimodal/src/autogluon/multimodal/constants.py

```diff
@@ -64,9 +64,37 @@
 SCORE = "score"
 LOGIT_SCALE = "logit_scale"
 
-# Metric
+# Metric for Object Detection
 MAP = "map"
 MEAN_AVERAGE_PRECISION = "mean_average_precision"
+MAP_50 = "map_50"
+MAP_75 = "map_75"
+MAP_SMALL = "map_small"
+MAP_MEDIUM = "map_medium"
+MAP_LARGE = "map_large"
+MAR_1 = "mar_1"
+MAR_10 = "mar_10"
+MAR_100 = "mar_100"
+MAR_SMALL = "mar_small"
+MAR_MEDIUM = "mar_medium"
+MAR_LARGE = "mar_large"
+DETECTION_METRICS = [
```

Comment:
Do we need tests for object detection metrics?

Reply:
Oh sorry for noticing this after the merge. I'll add the test to it in an existing one in next pr.